pyuntl/untl_structure.py

```diff
@@ -457,6 +458,48 @@ def generate_form_data(self, **kwargs):
         # Create the form object.
         return FormGenerator(**kwargs)
 
+    def make_hidden(self):
+        """Make an unhidden UNTL element into a hidden element."""
+        meta_hidden = False
+        for element in self.children:
+            if element.tag == 'meta' and element.qualifier == 'hidden':
+                meta_hidden = True
+                # Make the element hidden.
+                if element.content == 'False':
+                    element.content = 'True'
+        # Create a hidden  meta element if it doesn't exist.
+        if not meta_hidden:
+            hidden_element = PYUNTL_DISPATCH['meta'](qualifier='hidden', content='False')
```

Comment:
I think this should be `content='True'`, since this is the `make_hidden` method.

Reply:
yea..i initially had that idea, but since @vphill mentioned that we shall default it to being visible when we are creating a new Meta hidden record, I changed it to `False`.

Reply:
That thread is a bit confusing, but I think that in the context of calling `is_hidden`, when there is not a meta element with hidden qualifier, we default to visible. However, I think in the context of a method called `make_hidden`, we always want the result to be that it the record ends up hidden, whether or not the meta element with hidden qualifier exists or not.